### PR TITLE
Add paddle softplus op support.

### PR DESCRIPTION
**Pull request by huzhaoyX**
Signed-off-by: huzhaoyX <zhaoyangx.hu@intel.com>

### Details:
 - *[FrontEnd] [Paddle] Add paddle softplus op support.*

### Tickets:
CVS-69465 PPYolov2 requires softplus

### CODE REVIEW

File: ngraph/frontend/paddlepaddle/src/op/softplus.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    auto beta = node.get_attribute<float>("beta");
+    auto threshold = node.get_attribute<float>("threshold");
+    float supported_beta = 1.0;
+    float supported_threshold = 20.0;
+    const float EPSINON = 1e-6;
+
+    if (!(abs(beta - supported_beta) <= EPSINON) || !(abs(threshold - supported_threshold) <= EPSINON)) {
+        PDPD_OP_VALIDATION_CHECK(node, false, "only support beta==1.0 && threshold==20.0");
+    }
+    return node.default_single_output_mapping({std::make_shared<ngraph::opset7::SoftPlus>(data)}, {"Out"});
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/softplus.cpp:23]: better not use opset7 here. 
Please refer to op/layer_norm.cpp. It uses default_opset.hpp and its namespace to decide the opset.

File: ngraph/frontend/paddlepaddle/src/op/softplus.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset7.hpp>
```
Comment (ceciliapeng2011) [inline ngraph/frontend/paddlepaddle/src/op/softplus.cpp:5]: use default_opset.hpp please

**Review by ceciliapeng2011 — CHANGES_REQUESTED** [review]

**Review by ceciliapeng2011 — APPROVED** [review]

File: ngraph/frontend/paddlepaddle/src/op/softplus.cpp [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ngraph {
```
Comment (ilyachur) [inline ngraph/frontend/paddlepaddle/src/op/softplus.cpp:8]: Please use ov namespace

**Review by ilyachur — COMMENTED** [review]
Please rebase your branch to the latest master

File: ngraph/frontend/paddlepaddle/src/op/softplus.cpp (hunk 3 above)
Comment (ceciliapeng2011) [reply ngraph/frontend/paddlepaddle/src/op/softplus.cpp]: hi @ilyachur  do we have to change namespace from ngraph to openvino for all paddle frontend ops? or we simply change it for new developed code files?

If it is required for all files, I am going to do it in a separate PR.

**Comment by ceciliapeng2011** [comment]
@mvafin @ilyachur could you please review this PR? CI passed.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**